DateTime.spec.ts

```diff
@@ -106,14 +106,14 @@ describe("DateTime", () => {
 				expect(isoly.DateTime.localize(new Date(date[0])).split(" ")[1]).toEqual(date[1])
 			)
 		it("localize DateTime with locale", () => {
-			expect(isoly.DateTime.localize("2020-12-31T23:59:59.000Z", "en-US")).toEqual("01/01/2021, 12:59:59 AM")
+			expect(isoly.DateTime.localize("2020-12-31T23:59:59.000Z", "en-US")).toEqual("01/01/2021, 12:59:59 AM")
```

Comment:
So thosee are changed from `Space` to `Narrow No-Break Space`.
GitHub-tests seems to be fine (!?), however I get an error on my local machine running the tests.

Reply:
Does your local machine expect Narrow No-Break Space?